pepper-apis/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -54,7 +54,7 @@ private ConfigManager(Config cfg) {
      * @return the config for the app
      */
     public static synchronized ConfigManager getInstance() {
-        if (configManager == null) {
+        if (configManager == null && TYPESAFE_CONFIG_FILE != null) {
```

Comment:
When we're running CLI tools (like `AesUtil`), the first thing that runs is the SlackAppender constructor, which calls this `getInstance()` method to grab the config object. Somehow, the `static` block in this file doesn't get run, which means `TYPESAFE_CONFIG_FILE` variable is null. So we add null handling here. Idea here is: if there's no config file, we won't initialize the config manager, so callers need to handle null config managers.